habit-backend/users/userhelper.py

```diff
@@ -0,0 +1,15 @@
+import re
+    
+def ValidateEmail(email):
+    """
+    This method is used for validating email.
+    Parameters: email
+    Returns: Status
+    """
+
+    # Regular expression for email validity
+    regex = '^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$'
+    # Validate input email against regex
+    if re.search(regex, email):
+        return 'Valid'
```

Comment:
why not just return `True/False` ?

Reply:
Yeah will change. Boolean will be a better approach..

Reply:
Actually just ‘return re.search(regex, email)’ would work I guess